lua/wikis/commons/Condition.lua

```diff
@@ -60,6 +60,29 @@ function ConditionTree:toString()
 
 end
 
+---@enum lpdbComparator
+local Comparator = {
+	equals = '::',
+	notEquals = '::!',
+	greaterThan = '::>',
+	lessThan = '::<',
+	greaterThanOrEqualTo = '>::',
+	lessThanOrEqualTo = '<::'
+}
+Comparator.eq = Comparator.equals
+Comparator.neq = Comparator.notEquals
+Comparator.gt = Comparator.greaterThan
+Comparator.lt = Comparator.lessThan
+Comparator.ge = Comparator.greaterThanOrEqualTo
+Comparator.le = Comparator.lessThanOrEqualTo
```

Comment:
do we want to add some
`---@diagnostic disable-next-line: inject-field` here so the anno warnings shut up?